kyo-core/shared/src/main/scala/kyo/loops.scala

```diff
@@ -72,7 +72,7 @@ object Loops:
                     loop(next.input1, next.input2)
                 case kyo: Kyo[Output | Continue2[Input1, Input2], S] @unchecked =>
                     kyo.map {
-                        case next: Continue2[Input1, Input2] =>
+                        case next: Continue2[Input1, Input2] @unchecked =>
```

Comment:
I'm noticing some non-determinism with the compiler. Sometimes unchanged code starts reporting a warning it. It seems related to inlining.